lineman/app/components/thread_page/undecided_panel/undecided_panel.coffee

```diff
@@ -0,0 +1,15 @@
+angular.module('loomioApp').directive 'undecidedPanel', ->
+  scope: {proposal: '='}
+  restrict: 'E'
+  templateUrl: 'generated/components/thread_page/undecided_panel/undecided_panel.html'
+  replace: true
+  controller: ($scope, Records, CurrentUser) ->
+
+    $scope.undecidedPanelOpen = false
+
+    $scope.showUndecided = ->
+      $scope.proposal.fetchUndecidedMembers()
```

Comment:
We don't generally have fetch methods on models, so this is a bit of a departure from the convention.
Why not 
```
Records.proposals.fetchUndecidedMembers($scope.proposal)
```
?